src/renderer/services/deposit/utils.ts

```diff
@@ -0,0 +1,34 @@
+import BN from "bn.js";
+
```

Comment:
@mpetrunic should I move this util file to some kind of util folder because these are kind of general util methods (especially last three)